### PR TITLE
Add Control Plane to ecosystem page

### PR DESCRIPTION
**Pull request by igorchyts**
## Description
Add Control Plane to ecosystem page

## Reviewers

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by kfaseela** [comment]
Please sign the CLA

File: data/companies.yml [hunk 1]
```diff
@@ -97,6 +97,9 @@ providers:
   - name: "IMESH Platform"
     logo: "/logos/imesh.svg"
     url: "https://imesh.ai/imesh-istio-platform.html"
+  - name: "Control Plane"
+    logo: "/logos/controlplane.png"
```
Comment (ericvn) [inline data/companies.yml:101]: Would you make this a svg.

Comment (igorchyts) [reply data/companies.yml]: Fixed.

**Review by kfaseela — APPROVED** [review]

**Pull request merged**